<?xml version="1.0" encoding="UTF-8"?>
<testng-results ignored="12" total="13" passed="0" failed="1" skipped="0">
  <reporter-output>
  </reporter-output>
  <suite started-at="2021-08-21T19:18:07 IST" name="Smoke Tests" finished-at="2021-08-21T19:18:22 IST" duration-ms="15313">
    <groups>
      <group name="userAccounts.accountLogin">
        <method signature="UserAccounts.accountLogin()[pri:2, instance:com.momstore.UserAccounts@6b53e23f]" name="accountLogin" class="com.momstore.UserAccounts"/>
      </group> <!-- userAccounts.accountLogin -->
    </groups>
    <test started-at="2021-08-21T19:18:07 IST" name="SmokeTests" finished-at="2021-08-21T19:18:22 IST" duration-ms="15313">
      <class name="com.momstore.UserAccounts">
        <test-method is-config="true" signature="preTestRun()[pri:0, instance:com.momstore.UserAccounts@6b53e23f]" started-at="2021-08-21T19:18:05 IST" name="preTestRun" description="Pre Test Configurations" finished-at="2021-08-21T19:18:07 IST" duration-ms="1949" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- preTestRun -->
        <test-method is-config="true" signature="preTestRuns()[pri:0, instance:com.momstore.UserAccounts@6b53e23f]" started-at="2021-08-21T19:18:07 IST" name="preTestRuns" description="Pre Test Configurations" finished-at="2021-08-21T19:18:17 IST" duration-ms="10693" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- preTestRuns -->
        <test-method signature="accountLogin()[pri:2, instance:com.momstore.UserAccounts@6b53e23f]" started-at="2021-08-21T19:18:17 IST" name="accountLogin" description="Logging into the User Account" finished-at="2021-08-21T19:18:21 IST" duration-ms="4005" status="FAIL">
          <exception class="java.lang.AssertionError">
            <message>
              <![CDATA[expected [SignIn] but found [Sign In with Email]]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.AssertionError: expected [SignIn] but found [Sign In with Email]
at org.testng.Assert.fail(Assert.java:97)
at org.testng.Assert.assertEqualsImpl(Assert.java:136)
at org.testng.Assert.assertEquals(Assert.java:118)
at org.testng.Assert.assertEquals(Assert.java:575)
at org.testng.Assert.assertEquals(Assert.java:585)
at com.momstore.UserAccounts.accountLogin(UserAccounts.java:108)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.base/java.lang.reflect.Method.invoke(Method.java:566)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:134)
at org.testng.internal.TestInvoker.invokeMethod(TestInvoker.java:597)
at org.testng.internal.TestInvoker.invokeTestMethod(TestInvoker.java:173)
at org.testng.internal.MethodRunner.runInSequence(MethodRunner.java:46)
at org.testng.internal.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:816)
at org.testng.internal.TestInvoker.invokeTestMethods(TestInvoker.java:146)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:146)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:128)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1541)
at org.testng.TestRunner.privateRun(TestRunner.java:766)
at org.testng.TestRunner.run(TestRunner.java:587)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:384)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:378)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:337)
at org.testng.SuiteRunner.run(SuiteRunner.java:286)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:53)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:96)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1187)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1109)
at org.testng.TestNG.runSuites(TestNG.java:1039)
at org.testng.TestNG.run(TestNG.java:1007)
at org.apache.maven.surefire.testng.TestNGExecutor.run(TestNGExecutor.java:283)
at org.apache.maven.surefire.testng.TestNGXmlTestSuite.execute(TestNGXmlTestSuite.java:75)
at org.apache.maven.surefire.testng.TestNGProvider.invoke(TestNGProvider.java:120)
at org.apache.maven.surefire.booter.ForkedBooter.invokeProviderInSameClassLoader(ForkedBooter.java:384)
at org.apache.maven.surefire.booter.ForkedBooter.runSuitesInProcess(ForkedBooter.java:345)
at org.apache.maven.surefire.booter.ForkedBooter.execute(ForkedBooter.java:126)
at org.apache.maven.surefire.booter.ForkedBooter.main(ForkedBooter.java:418)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.AssertionError -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- accountLogin -->
        <test-method is-config="true" signature="postTestRuns()[pri:0, instance:com.momstore.UserAccounts@6b53e23f]" started-at="2021-08-21T19:18:22 IST" name="postTestRuns" description="Post Test configurations" finished-at="2021-08-21T19:18:22 IST" duration-ms="240" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- postTestRuns -->
        <test-method is-config="true" signature="postTestRun()[pri:0, instance:com.momstore.UserAccounts@6b53e23f]" started-at="2021-08-21T19:18:22 IST" name="postTestRun" description="Final finish configurations" finished-at="2021-08-21T19:18:30 IST" duration-ms="7828" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- postTestRun -->
      </class> <!-- com.momstore.UserAccounts -->
    </test> <!-- SmokeTests -->
  </suite> <!-- Smoke Tests -->
</testng-results>
